frontend/src/app/components/wp-fast-table/builders/drag-and-drop/drag-drop-handle-render-pass.ts

```diff
@@ -0,0 +1,34 @@
+import {Injector} from '@angular/core';
+import {WorkPackageTableColumnsService} from '../../state/wp-table-columns.service';
+import {PrimaryRenderPass, RowRenderInfo} from '../primary-render-pass';
+import {DragDropHandleBuilder} from "core-components/wp-fast-table/builders/drag-and-drop/drag-drop-handle-builder";
+import {WorkPackageTable} from "core-components/wp-fast-table/wp-fast-table";
+
+export class DragDropHandleRenderPass {
+
+  public wpTableColumns = this.injector.get(WorkPackageTableColumnsService);
+
+  // Drag & Drop handle builder
+  protected dragDropHandleBuilder = new DragDropHandleBuilder(this.injector);
+
+  constructor(public readonly injector:Injector,
+              private table:WorkPackageTable,
+              private tablePass:PrimaryRenderPass) {
+  }
+
+  public render() {
+    this.tablePass.renderedOrder.forEach((row:RowRenderInfo, position:number) => {
+      // We only care for rows that are natural work packages and are not relation sub-rows
+      if (!row.workPackage || row.renderType ===  'relations') {
+        return;
+      }
+
+      const handle = this.dragDropHandleBuilder.build(row.workPackage);
+
+      if (handle) {
+        const element:HTMLElement = this.tablePass.tableBody.children[position] as HTMLElement;
+        element.replaceChild(handle, jQuery(element).find('.__internal-sorthandle')[0]);
```

Comment:
I think it would be okay in this case to assert the column is the first one and simply use `element.replaceChild(handle, element.firstElementChild`. If you disagree, we can at least use `element.replaceChild(handle, element.querySelector('.__internal-sorthandle')`

Reply:
👍